<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_grass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Grass block</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_grass_snow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Snowed grass block</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_dirt_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirt path</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_mycelium">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mycelium</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_podzol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Podzol</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_mycelium_snow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Snowed mycelium</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_podzol_snow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Snowed podzol</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_crimson_nylium">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Crimson nylium</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_warped_nylium">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Warped nylium</string>
   </property>
  </widget>
  <widget class="QLabel" name="grass_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="snow_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dirt_path_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mycelium_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="podzol_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="grass_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="choosefile">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="crimson_nylium_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warped_nylium_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dirt_path_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="choosefile">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mycelium_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="choosefile">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="podzol_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="choosefile">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="crimson_nylium_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warped_nylium_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="snow_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="grass_side_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="grass_side_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dirt_path_side_pathbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirt_path_side_pathview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
